tests/test_cli_download_files.py

```diff
@@ -123,6 +123,24 @@ def test_download_files_https_requests(mocker):
         os.remove(test_file)
 
 
+def test_download_files_download_engine(mocker):
```

Comment:
I see troubles with Python-2.7 locally. The `importlib-resources` was installed but is not Python-2.7 friendly, since it uses f-strings. You need to pin some package versions in `setup.py` conditionally, so that they would not be installed for Python-2.7.  Here is a test case to try out locally: 

```console
$ tox -e py27
...
  File "/home/tibor/private/project/opendata/src/cernopendata-client/.eggs/importlib_resources-0.0.0-py2.7.egg/importlib_resources/readers.py", line 77
    raise FileNotFoundError(f'{self} is not a file')
                                                  ^
...
  File "/home/tibor/private/project/opendata/src/cernopendata-client/.tox/py27/lib/python2.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'importlib-resources>=1.0' distribution was not found and is required by virtualenv
ERROR: InvocationError for command /home/tibor/private/project/opendata/src/cernopendata-client/.tox/py27/bin/python setup.py test (exited with code 1)
```

Reply:
I tested the `$ tox -e py27` in a `python2.7 env`. It worked fine without warnings.
[log.txt](https://github.com/cernopendata/cernopendata-client/files/5800942/log.txt)

Reply:
Solved: this was observed with OS native `python3-tox` package. Using `tox` from PyPI fixes the troubles.